includes/admin/customers/customer-actions.php

```diff
@@ -231,10 +232,25 @@ function edd_add_customer_email( $args ) {
 				);
 
 			} else {
+				// Find out if this email is taken by a user or a customer and then include a link to the correct admin view.
+				$user_type = 'customer';
+				$user = new EDD_Customer( $email );
+
+				if( empty($user->email) ){
+
+					$user = get_user_by( 'email', $email );
+					$user_type = 'user';
+				}
+
+				$link = (($user_type === 'customer') ?
+				admin_url( 'edit.php?post_type=download&page=edd-customers&view=overview&id=' . $user->id ) :
+				admin_url( 'user-edit.php?user_id=' . $user->id ));
 
 				$output = array(
 					'success' => false,
-					'message' => __( 'Email address is already associated with another customer.', 'easy-digital-downloads' ),
+					'message' => __( 'Email address is already associated with another', 'easy-digital-downloads' ),
```

Comment:
`message` here has been modified but is now incomplete:
`Email address is already associated with another` is not a complete sentence.